cli/test/lib/core/ping.js

```diff
@@ -0,0 +1,59 @@
+const test = require('ava');
+const FakeApi = require('../../utils/FakeApi');
+
+// Mock private function that uses Date.now to
+// get consistent return value
+const rewire = require('rewire');
```

Comment:
This is kind of hacky. I might be able to find a way to do this with `sinon`, but didn't see a clear way to do this in their docs.